_posts/francois_gerthoffert/2017-08-21-Infrastructure-procurement-in-large-scale-genomic-projects.md

```diff
@@ -0,0 +1,163 @@
+
+The initial stage of a RFSQ has similarities with a RFP in the sense that it contains contractual details and, in our situation, relied on a "simulated" proposal to evaluate vendors. Following the RFSQ, subsequent procurement is executed through multiple RFQ phases.
+
+After award the RFSQ is limited to an initial set of vendors, which which a contract was signed. But at any point in time, we have the flexibility to initiate a RFP and run purchases outside of the RFSQ framework (as long as the RFP is executed following corresponding rules).
+
+RFSQ are very common for maintenance activities (such as gardening, electrical work), less for IT Infrastructure projects. It was suggested to us by our procurement department as we were discussing the project's procurement approach.
+
+# Evaluation
+
+While preparing a request using one of the methods detailed above, it is critical to identify clear evaluation factors and assign each of them a "weight" in order to establish a score. This allows a fair evaluation of the vendors based on transparent criteria. The evaluation could for example range from 1 to 5, while the weight could be a percentage, giving you a total score to be used while comparing vendors.
+
+| Category      | Evaluation  | Weight  |
+|---|---|---|
+|Price          | 1 to 5 | 50%  |
+|Delivery Time  | 1 to 5 | 20%  |
+|Warranty Period| 1 to 5 | 30%  |
+
+The selected vendor will therefore be the one with the closest score to 100.
+
+Establishing the evaluation factors up-front is also a technique to structure your requests and communicate to vendors how their answer should be formatted. The more technical the request is (RFQ in particular), the more precise the evaluation factors are going to be, and the last thing you want when comparing proposals is to spend hours dissecting a proposal to identify factors impacting the scoring.
+
+Less technical requests (RFIs for example), could be evaluated on additional criteria such as:
+
+ - Quality of the technical response
+ - Technical expertise of the vendor's personnel (based on their bios, certifications ...)
+ - Experience of the vendor in delivering similar projects
+
+Keep in mind, while preparing the request and evaluating responses that the less technical the request and evaluation criteria is, the harder it becomes to evaluate and justify such an evaluation. So chose your evaluation criteria wisely to serve your project's purpose and facilitate the procurement process, in particular if a vendor decide to challenge the award decision.
+
+# Our strategy
+
+As mentioned above, the RFSQ appeared to be the ideal method for Collaboratory procurement, we used the initial purchase as the baseline to build the RFSQ and to pre-select vendors.
```

Comment:
As mentioned above, the RFSQ appeared to be the ideal method for Collaboratory procurement, and we used the initial purchase as a baseline to build the RFSQ and to pre-select vendors.